README.org

```diff
@@ -66,7 +66,12 @@ By enabling “Use Unicode Symbols” from the Settings menu, Casual Avy will us
 | Next     | Next     | ↓       |
 
 ** Imenu (index) Support
-The Emacs [[https://www.gnu.org/software/emacs/manual/html_node/emacs/Imenu.html][Imenu]] (index menu) feature offers a way to navigate to a major definition in a file, provided that the current mode supports it. As Imenu behavior is closely related to Avy, support for it is provided here as the menu item labeled "(i) Index".
+The Emacs [[https://www.gnu.org/software/emacs/manual/html_node/emacs/Imenu.html][Imenu]] (index menu) feature offers a way to navigate to a major definition in a file, provided that the current mode supports it. As Imenu behavior is closely related to Avy, support for it is provided here as the menu item labeled "(i) Index". Be sure to enable the index menu to get the benefits:
```

Comment:
Replace "Be sure to enable the index menu to get the benefits:" with the following text. 

"The following configuration code turns on index menus for Markdown, Org, Makefile, and programming language modes."